charts/qdrant/values.yaml

```
@@ -20,6 +20,7 @@ service:
   type: ClusterIP
   additionalLabels: {}
   annotations: {}
+  loadBalancerIP: {}
```

```suggestion
  loadBalancerIP: ""
```

`loadBalancerIP` is a string in the [ServiceSpec](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/#ServiceSpec). Thus the default value should be an empty string here.